pkg/apb/svc_acct.go

```diff
@@ -86,11 +86,7 @@ func (s *ServiceAccountManager) createResources(rFilePath string, namespace stri
 	return nil
 }
 
-func (s *ServiceAccountManager) writeResourceFile(
-	handle string,
-	svcAcctM *map[string]interface{},
-	roleBindingM *map[string]interface{},
-) (string, error) {
+func (s *ServiceAccountManager) writeResourceFile(handle string, svcAcctM *map[string]interface{}, roleBindingM *map[string]interface{}) (string, error) {
```

Comment:
I like the long lines better personally but it seems I am outvoted here. I'll change it back and update my environment to alert me.